src/Resolvers/Microsoft.DotNet.NativeWrapper/Interop.cs

```diff
@@ -152,18 +152,6 @@ public static class Unix
                 string exe_dir,
```

Comment:
Nit: Not related to this change - the other hostfxr imports here can probably also be unified between Windows and Unix.

Reply:
I'm going to leave this for now to minimize the scope of this PR, but I'll keep this in mind for future changes.